wiki/Beatmap/Pattern/Mania/en.md

```diff
@@ -0,0 +1,27 @@
+---
+stub: true
+tags:
+  - patterns
+  - osu!mania
+---
+
+# osu!mania Patterns
+
+A **pattern** is a small set of [hit objects](/wiki/Hit_object) within a [beatmap](/wiki/Beatmap) arranged in a meaningful way. In osu!mania, this can be done with either **notes**, **long notes**, or a combination of the two.
```

Comment:
Maybe add the rice/noodles thing here?

Reply:
I don't really know how to integrate that into this sentence without cluttering it too much, any ideas?

Reply:
Maybe `**notes** (rice)` and similarly to LNs. Not sure if this is needed though.

Reply:
Think this would just convolute this section, it doesn't seem all that necessary to mention both here when the goal is to keep the explanation simple